SingularityUI/app/views/expandableTableSubview.coffee

```diff
@@ -25,6 +25,7 @@ class ExpandableTableSubview extends View
 
     initialize: ({@collection, @template}) ->
         @listenTo @collection, 'sync', @render
+        @listenTo @collection, 'reset', @render
```

Comment:
I'm not sure if we can drop sync completely here. I imagine reset should only fire when we call `collection.reset`, so other views using expandableTableSubview may not render.